### PR TITLE
Project week 2, Ingrid H

### PR DESCRIPTION
**Pull request by ingrid2019-ism**
I think I managed to finish the requirements but I didn´t had time to start on any of the stretch goals.
I struggled a lot with the sizes of the elements and to get them to fit together. I ´m also still confused by the fact that the site looks different every time I reopen it in the browser...

### CODE REVIEW

**Comment by ingrid2019-ism** [comment]
I think it was because I forgott to "unclick" the button for responsive 
in the inspector tool. I don´t know, maybe I was just overworked and 
confused XD

------ Originalmeddelande ------
Från: "Erica Glimsholt" <notifications@github.com>
Till: "Technigo/project-news-site" 
<project-news-site@noreply.github.com>
Kopia: "ingrid2019-ism" <ingrid.hammargren@gmail.com>; "Author" 
<author@noreply.github.com>
Skickat: 2020-01-27 21:45:18
Ämne: Re: [Technigo/project-news-site] Project week 2, Ingrid H (#33)

File: code/style.css [hunk 1]
```diff
@@ -1,4 +1,193 @@
...
+  margin-left: 10px;
+  margin-right: 10px;
+  margin-bottom: 10px;
+  padding-top: 40px;
+}
+
+.big-news {
+  background-color: #d3d3da ;
+  display: flex;
+
```
Comment (ericaglimsholt) [inline code/style.css]:
```suggestion
```

File: code/style.css [hunk 2]
```diff
@@ -1,4 +1,193 @@
...
+}
+
+
+.series-of-articles {
+  background-color:  #f4f4fc;
+  display: flex;
+
```
Comment (ericaglimsholt) [inline code/style.css]:
```suggestion
```

File: code/style.css [hunk 3]
```diff
@@ -1,4 +1,193 @@
...
+  border: 1px solid #ddd;
+  border-radius: 15px;
+  padding: 5px;
+  width: 80%;
+}
+
+.big-news-cat {
+  background-color: darkgray;
+  width: 50%;
+
```
Comment (ericaglimsholt) [inline code/style.css]:
```suggestion
```

File: code/style.css [hunk 4]
```diff
@@ -1,4 +1,193 @@
...
+.big-news-cat {
+  background-color: darkgray;
+  width: 50%;
+ 
+}
+
+ .other-news {
+   display: flex;
+   flex-direction: column;
+  align-items: center;
```
Comment (ericaglimsholt) [inline code/style.css]:
```suggestion
  align-items: center;
```

File: code/style.css [hunk 5]
```diff
@@ -1,4 +1,193 @@
...
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: solid; color: black;
+  border-radius: 15px;
+  padding: 10px;
+  margin: 15px;
+  width: 70%;
+
```
Comment (ericaglimsholt) [inline code/style.css]:
```suggestion
```

**Pull request closed without merging**